khl/client.py

```diff
@@ -363,3 +367,27 @@ async def fetch_blocked_friends(self) -> List[Friend]:
 
     async def start(self):
         await asyncio.gather(self.handle_pkg(), self.gate.run(self._pkg_queue))
+
+    async def view_message(self, msg_id: str) -> Dict:
+        """get message detail by message id"""
+        return await self.gate.exec_req(api.Message.view(msg_id))
+
+    async def list_messages(self,
+                            channel: Union[Channel, str],
+                            page_size: int = None,
+                            pin: int = None,
+                            flag: MessageFlagModes = None,
+                            msg_id: str = None) -> Dict:
+        """list the messages by guild id"""
+        target_id = unpack_id(channel)
+        params = {'target_id': target_id}
+        if page_size is not None:
+            params['page_size'] = page_size
+        if pin is not None:
+            params['pin'] = pin
+        if flag is not None:
+            params['flag'] = unpack_value(flag)
+        if msg_id is not None:
+            params['msg_id'] = msg_id
+        params['page'] = 0
+        return await self.gate.exec_req(api.Message.list(**params))
```

Comment:
marked duplicated by pylint here, a small trick:
```suggestion
        if isinstance(channel, str):
            channel = PublicChannel(id=channel, _gate_=self.gate)
        return await channel.list_messages(page_size,pin,flag,msg_id)
```